app/components/Comments/CommentView.tsx

```diff
@@ -27,21 +51,87 @@ const CommentView = (props: Props) => {
     contentTarget,
     style,
     displayTitle = true,
-    newOnTop = false,
     contentAuthors,
+    newOnTop,
   } = props;
   const commentFormProps = {
     contentTarget,
   };
-  const tree = generateTreeStructure(comments);
+  const [ordering, setOrdering] = useState<Option>(orderingOptions[0]);
+  const [sortedComments, setSortedComments] = useState<Comment[]>(comments);
```

Comment:
Don't use a useState for this, just calculate based on `ordering` in the component. I think best practice is to avoid useState if possible.